samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/bundle/DeliveryMessages_eu.properties

```diff
@@ -570,3 +570,10 @@ savedMessage=Work is being saved.
 subMessage=Work is being submitted.
 hideMessage=Hide Time Remaining
 showMessage=Show Time Remaining
+
+# S2U-21 Safe Exam Browser
+begin_assessment_msg_seb_required=Este examen requiere usar Safe Exam Browser. Por favor, aseg\u00farate que Safe Exam Browser est\u00e1 instalado y haz click en el bot\u00f3nn ejecutar.
+seb_download_configuration=Descargar Configuraci\u00f3n
+seb_download_seb=Descargar Safe Exam Browser
+seb_launch_seb=Ejecutar Safe Exam Browser
+seb_quit_browser=Salir de Safe Exam Browser
```

Comment:
These 5 strings are the ones missing the EU translation. @PabloSanRoman

Reply:
@mpellicer : 

+ begin_assessment_msg_seb_required= Azterketa honek behar du erabili “Azterketa seguruko nabigatzailea” (Safe Exam Browser). Ziurtatu Safe Exam Browser instalatuta dagoela eta egin klik exekutatu botoian.
+ seb_download_configuration=Deskargatu konfigurazioa
+ seb_download_seb=Deskargatu Safe Exam Browser
+ seb_launch_seb=Exekutatu Safe Exam Browser
+ seb_quit_browser=Atera nabigatzaile segurutik (Safe Exam Browser)

Reply:
Thanks a million!